qiita_db/reference.py

```diff
@@ -23,6 +79,7 @@ class Reference(QiitaObject):
     sequence_fp
     taxonomy_fp
     tree_fp
+    sortmerna_indexed_db
```

Comment:
I don't believe it is correct to assume that all sequence databases will (or can) have a sortmerna index. For instance, I do not believe `indexdb_rna` will work with protein sequence.